run_server.py

```diff
@@ -105,7 +105,7 @@ def usage(fd=sys.stdout):
 except (ValueError, IndexError):
 	port = 0
 
-if host == None or port == None or port <= 0:
+if None in [host, port] or port <= 0:
```

Comment:
Forgot this one, this should be using `is`. See my comment on `development/networkclient.py`.